CONTRIBUTING.md

```diff
@@ -1,13 +1,31 @@
 # Contributing to GCRCatalogs
 
+## Preparing catalog files
+
+Consider the following when you prepare the catalog files that you plan to add to `GCRCatalogs`. 
+
+- File format: We strongly recoommend that you store the file in the Apache Parquet format.
```

Comment:
is it worthwhile to point out that other formats are also supported while also making it clear that there is a strong preference for parquet?  
Maybe:
"While GCRCatalogs can support any file format, we strongly recommend that your store..."